manifests/init.pp

```diff
@@ -151,6 +157,17 @@
       key     => $certs::apache::apache_key,
       ca_cert => $certs::ca_cert,
       require => Class['certs::apache'],
+      port    => $docker_registry_port,
+    }
+  }
+
+  if $pulp_master {
+    $check_file = file('/usr/sbin/foreman-rake', '/dev/null')
```

Comment:
What is this supposed to achieve? Afaik the [file function](https://docs.puppet.com/puppet/latest/reference/function.html#file) is for using the puppet fileserver and I don't think you're doing what you intended to do.